src/pages/Contests/Contests.js

```diff
@@ -0,0 +1,307 @@
+import React, { useState, useEffect } from 'react'
+// import Loading from '../logreg/loading.jsx'
+// import Navbar from '../../components/Header/Navbar'
+// import FooterSmall from '../../components/Footer/FooterSmall'
+import queryString from 'query-string'
+import update from 'immutability-helper'
+import AccordionCom from '../../components/AccordionCom'
+
+import 'antd/dist/antd.css'
+import '../../styles/Contests/contests.css'
+import { Button, Col, Switch, Input, Row } from 'antd'
+import { CloseOutlined, CheckOutlined } from '@ant-design/icons'
+import Modal_comm from './Modal_Com'
+
+function ContestPage({info, queryStr}) {
+  //queryStr is current url string
+  const [datap, setDatap] = useState(queryStr.replace(/;/g, '&'))
+
+  // search box
+  const {Search} = Input;
+
+  //Parsing into json
+  const [queryDefault, setQueryDefault] = useState(
+    queryString.parse(
```

Comment:
Try finding an alternative to queryString as we are using this dependency just for filters.